resource/js/scripts.js

```diff
@@ -210,7 +210,8 @@ function loadLimitedResults(parameters) {
   clearResultsAndAddSpinner();
   $.ajax({
     data: parameters,
-    success : function(data) {
+    complete : function(jqXHR, textStatus) {
```

Comment:
This callback will be executed in any case, both for successful and unsuccesful requests.